ai/ollama/v0/config/tasks.json

```diff
@@ -0,0 +1,337 @@
+            "number"
+          ],
+          "instillUIOrder": 5,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "Temperature",
+          "type": "number"
+        },
+        "top-k": {
+          "default": 10,
+          "description": "Top k for sampling",
+          "instillAcceptFormats": [
+            "integer"
+          ],
+          "instillUIOrder": 5,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "Top K",
+          "type": "integer"
+        }
+      },
+      "required": [
+        "prompt",
+        "model-name"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Output",
+      "instillUIOrder": 0,
+      "properties": {
+        "text": {
+          "description": "Model Output",
+          "instillUIOrder": 0,
+          "instillFormat": "string",
+          "instillUIMultiline": true,
+          "title": "Text",
+          "type": "string"
+        },
+        "usage": {
+          "$ref": "#/$defs/usage"
+        }
+      },
+      "required": [
+        "text"
+      ],
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_TEXT_EMBEDDINGS": {
+    "instillShortDescription": "Turn text into a vector of numbers that capture its meaning, unlocking use cases like semantic search.",
+    "description": "An embedding is a list of floating point numbers that captures semantic information about the text that it represents.",
+    "input": {
+      "instillUIOrder": 0,
+      "properties": {
+        "model-name": {
+          "example": "snowflake-arctic-embed:22m",
+          "description": "The OSS model to be used, check https://ollama.com/library for list of models available",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUIOrder": 0,
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "title": "Model Name",
+          "type": "string"
+        },
+        "text": {
+          "description": "The text",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUIMultiline": true,
+          "instillUIOrder": 1,
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "title": "Text",
+          "type": "string"
+        }
+      },
+      "required": [
+        "text",
+        "model-name"
+      ],
+      "instillEditOnNodeFields": [
+        "text",
+        "model-name"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "instillUIOrder": 0,
+      "properties": {
+        "embedding": {
+          "instillFormat": "array:number",
+          "items": {
+            "instillFormat": "number",
+            "title": "Embedding",
```

Comment:
nit
`"title": "Embedding",`
Items are vectors?

Reply:
I found that I missed in all previous PR..

Reply:
By definition embedding are vectors, but embedding means it contains some low dimension features from the text, thus `Embedding` will be a better name.

Reply:
Actually, the item in an embedding is just a number, rather than an actual embedding. I think we don't even need the title here.

Reply:
Ok, sure. I will remove the title.

Reply:
Fixed